creator-node/src/premiumContent/helpers.ts

```diff
@@ -1,5 +1,63 @@
 import { signatureHasExpired } from '../apiSigning'
 import { PremiumContentSignatureData, PremiumContentType } from './types'
+import { getRegisteredDiscoveryNodes } from '../utils/getRegisteredDiscoveryNodes'
+import { Redis } from 'ioredis'
+import type Logger from 'bunyan'
+
+const models = require('../models')
+
+const PREMIUM_CONTENT_SIGNATURE_MAX_TTL_MS = 6 * 60 * 60 * 1000 // 6 hours
+
+export async function isCIDForPremiumTrack(cid: string): Promise<
+  | {
```

Comment:
nit - lets add these return types into the types.ts file since we do this for other premium content types

Reply:
those other types are used in multiple places but this one is not, so I kinda like it being colocated. lmk if you feel strongly